.github/workflows/quic_matrix.json

```diff
@@ -1,10 +1,8 @@
 [
-    { "env": "azure", "os": "ubuntu-20.04", "arch": "x64", "tls": "openssl",  "io": "epoll" },
-    { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "iocp" },
-    { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "xdp" },
-    { "env": "azure", "os": "windows-2022", "arch": "x64", "tls": "schannel", "io": "wsk" },
```

Comment:
@ProjectsByJackHe are you Ok with removing these until we get the 1ES pools working?

Reply:
FYI I already added code in manage-azure-machines.ps1 to filter out the Azure VMs. This is unnecessary.

Reply:
![image](https://github.com/microsoft/netperf/assets/43735701/cd7dfd66-70fc-4d4a-b6f8-212693c8304e)

Reply:
@ami-GS can you remove these changes? then I can approve